lib/main.dart

```diff
@@ -16,7 +17,7 @@ import 'package:wordpress_api/wordpress_api.dart';
 void main() async {
   Logger.root.level = Level.INFO;
   Logger.root.onRecord.listen((record) {
-    print(
+    debugPrint(
```

Comment:
`debugPrint` will not attempt to print on production builds, as opposed to `print`